tools/publisher/src/subcommand/upgrade_runtime_crates_version.rs

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::fs::Fs;
+use anyhow::{anyhow, bail, Context};
+use clap::Parser;
+use regex::Regex;
+use std::path::{Path, PathBuf};
+
+#[derive(Parser, Debug)]
+pub struct UpgradeRuntimeCratesVersionArgs {
+    /// The version of runtime crates you want the code generator to use (e.g. `0.52.0`).
+    #[clap(long)]
```

Comment:
nit: Perhaps this should just be a positional arg, lest it be misconstrued as the version of a part of the CLI (?)